### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by waynemee** [community/community#114387] [Repositories]
I'm trying to install the modded Ubuntu I get to the part git clone git://github.com/modded-ubuntu/modded-ubuntu.git

and it just times out is there anymore links.thanks

**Comment by abdulhdr1** [comment]
Hey, it seems like there may be an issue with GitHub's `git://` protocol, I'd suggest using HTTPS or SSH instead:
`git clone https://github.com/modded-ubuntu/modded-ubuntu.git`
or
`git clone git@github.com:modded-ubuntu/modded-ubuntu.git`

**Comment by Santa-Dev-Hub** [comment]
i think you are writing the git clone command wrong!
try writing like this
git clone "https://github.com/modded-ubuntu/modded-ubuntu.git"
or
git clone "git://github.com/modded-ubuntu/modded-ubuntu.git"
this will definitely work and it it dosen't then do reply me back with the issue i will be happy to help!

**Comment by waynemee** [comment]
Hi thanks for your reply I have tried them but getting no file and forgot
to mention its on Google pixel 8 phone.

**Reply by abdulhdr1** [reply]
How are you using to access the CLI? It's an app or via `adb`? 
Also if you can add what you are getting when using the `--verbose` flag to the command it would be great.

**Comment by waynemee** [comment]
Trying to install modded Ubuntu from GitHub with termux .

   -

   First Clone the Repository & Run the setup File
   - pkg update -y && pkg upgrade -y
      - pkg install git wget -y
      - git clone git://github.com/modded-ubuntu/modded-ubuntu.git
      - cd modded-ubuntu
      - bash setup.sh
   -

   Then Restart your Termux & Type the following commands
   - ubuntu
      - bash user.sh
   -

   Type your ubuntu root username. Must be lowercase & no space included.
   -

   Then Again Restart your Termux & Type the following commands
   - ubuntu
      - bash gui.sh

**Reply by abdulhdr1** [reply]
the docs show `git clone --depth=1 https://github.com/modded-ubuntu/modded-ubuntu.git` instead of `git clone git://github.com/modded-ubuntu/modded-ubuntu.git`, could you give it a go?

**Comment by waynemee** [comment]
It's saying no such file or directory.
